src/System.Net.Sockets/src/System/Net/Sockets/_ReceiveMessageOverlappedAsyncResult.cs

```diff
@@ -10,200 +10,56 @@
 
 namespace System.Net.Sockets
 {
-    //
```

Comment:
From the title, I assumed that this is the final form. 
This answers both this and the above statement regarding Overlapped.